rho/utilities.py

```diff
@@ -113,10 +113,10 @@
                       'subman.virt.uuid',
                       'subman.has_facts_file')
 
-JBOSS_FACTS_TUPLE = ('jboss.running-versions',)
+JBOSS_FACTS_TUPLE = ('jboss.eap.running-versions',)
```

Comment:
Updates needed in `doc/source/command_syntax_usage.rst` with changed fact names.